go/oasis-test-runner/scenario/e2e/stake_cli.go

```diff
@@ -63,6 +63,9 @@ const (
 
 	// Testing escrow account public key (hex-encoded).
 	escrowPubkeyHex = "6ea5328f943ef6f66daaed74cb0e99c3b1c45f76307b425003dbc7cb3638ed35"
+
+	// Oasis Protocol Foundation's public gRPC endpoint.
+	publicGrpcAddr = "grpc.oasis.dev:443"
```

Comment:
Should we disable this test by default? Otherwise this will make CI fail if the public endpoint is unavailable.

Reply:
oasis-core intentionally is network instance neutral.  I'm not sure this feature warrants changing that policy...

Reply:
This is a test, not some default.

Reply:
I went ahead and removed the test since it would complicate things and is not really that crucial.